plugins/dialog/plugin.js

```diff
@@ -3031,19 +3031,71 @@ CKEDITOR.DIALOG_STATE_BUSY = 2;
 			cssLengthRegex = /^(((\d*(\.\d+))|(\d*))(px|em|ex|in|cm|mm|pt|pc|\%)?)?$/i,
 			inlineStyleRegex = /^(\s*[\w-]+\s*:\s*[^:;]+(?:;|$))*$/;
 
+		/**
+		 * {@link CKEDITOR.dialog Dialog} `OR` logical value indicates
+		 * relation between validation functions.
+		 *
+		 * @readonly
+		 * @property {Number} [=1]
+		 * @member CKEDITOR
+		 */
 		CKEDITOR.VALIDATE_OR = 1;
+
+		/**
+		 * {@link CKEDITOR.dialog Dialog} `AND` logical value indicates
+		 * relation between validation functions.
+		 *
+		 * @readonly
+		 * @property {Number} [=2]
+		 * @member CKEDITOR
+		 */
 		CKEDITOR.VALIDATE_AND = 2;
 
+		/**
+		 * The namespace with dialog helper validation functions.
+		 *
+		 * @class
+		 * @singleton
+		 */
 		CKEDITOR.dialog.validate = {
+			/**
+			 * Performs validation functions composition.
+			 *
+			 * ```javascript
+			 * CKEDITOR.dialog.validation.functions(
+			 * 	CKEDITOR.dialog.validation.notEmpty( 'Value is required.' ),
+			 * 	CKEDITOR.dialog.validation.number( 'Value is not a number.' ),
+			 * 	'error!'
+			 * );
+			 * ```
+			 *
+			 * @param {Mixed...} arguments
+			 * Accepts multiple functions which will be composed into a single validation function.
+			 *
+			 * ```javascript
+			 * CKEDITOR.dialog.validation.functions( validator1, validator2 );
+			 * ```
+			 *
+			 * Also you can pass two optional parameters:
```

Comment:
If these values are really accepted at the very end of the parameters list, then probably it'll be better to list three parameters, analogically to https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_tools.html#method-extend

Reply:
In such case `Mixed...` will become `Function...`